source/api_documentation/transactions_api/advertiser_user.rst

```diff
@@ -25,7 +25,7 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
     - Returns a JSON array of transaction objects. ``https://mynetwork.invoca.net/api/@@TRANSACTION_API_VERSION/advertisers/transactions/33.json``
 
 
-.. include:: intro.rst
+.. include:: _intro.rst
```

Comment:
Come to think of it, I think there is code in conf.py that looks specifically for files starting with underscores and identifies as something it needs to templatize. Might be worth looking into avoid unnecessary processing.